src/toil/batchSystems/abstractGridEngineBatchSystem.py

```diff
@@ -428,7 +434,12 @@ def killBatchJobs(self, jobIDs):
         for jobID in jobIDs:
             self.killQueue.put(jobID)
         while jobIDs:
-            killedJobId = self.killedJobsQueue.get()
+            try:
+                killedJobId = self.killedJobsQueue.get(timeout=10)
+            except queue.Empty:
```

Comment:
It looks like we already had `from queue import Empty, Queue`, so we should probably use the shorter name:
```suggestion
            except Empty:
```